components/Theme.tsx

```diff
@@ -0,0 +1,19 @@
+import {useEffect, useState} from "react";
+
+const Theme = (theme?: string) => {
+  const [isReady, setIsReady] = useState(false);
+  useEffect( () => {
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
  useEffect(() => {
```